doc/nrf/working_with_nrf/nrf91/nrf9160.rst

```diff
@@ -113,27 +113,26 @@ Complete the following steps to provision the certificate:
 #. If you have already inserted the SIM card into your DK, remove it before you continue.
 #. Connect the DK to the computer with a micro-USB cable, and turn it on.
 #. Click :guilabel:`Select device` and select the DK from the drop-down list.
-   You can identify the nRF9160 DK by the fact that it has three COM ports.
 
    .. figure:: images/ltelinkmonitor_selectdevice.png
-      :alt: LTE Link Monitor - COM ports
+      :alt: LTE Link Monitor - Select the nRF9160 DK
 
-      LTE Link Monitor - COM ports
+      LTE Link Monitor - Select the nRF9160 DK
 
-   If the three COM ports are not visible, press ``Ctrl+R`` in Windows or ``command+R`` in macOS to restart the Programmer application.
+   If the DK is not visible, press ``Ctrl+R`` in Windows or ``command+R`` in macOS to restart the Programmer application.
 
    The drop-down text changes to the type of the selected device, with the SEGGER ID below the name.
 
-#. Click :guilabel:`TERMINAL` in the navigation bar to switch to the terminal view.
+#. Click :guilabel:`TERMINAL` in the navigation bar to switch to the terminal view if it is not already selected.
 
    .. figure:: images/ltelinkmonitor_navigationterminal.png
       :alt: LTE Link Monitor - Terminal
 
       LTE Link Monitor - Terminal
 
-#. Enter ``AT+CFUN=4`` in the text field for AT commands and click :guilabel:`Send`.
+#. Enter ``AT+CFUN=4`` in the text field for AT commands and click :guilabel:`Send`, or click the :guilabel:`AT+CFUN=4` button.
```

Comment:
I think AT+CFUN=4 isn't one of the default buttons, at least in my instance of LTE Link Monitor. 

I actually didn't know that you could create your own AT command shortcuts with drag-and-drop until just now! So much more convenient. It always annoyed me that the button to turn the modem off wasn't there.

Reply:
Huh, I wonder if they used to be and it's bringing them along each time I update. Haven't done a clean install since first installing, I think. Will remove mention of it, in which case it doesn't make much sense to mention for the other command either.